src/dashboard/src/Layout/NotificationBox.tsx

```diff
@@ -30,29 +31,58 @@ const usePaperStyle = makeStyles(theme => createStyles({
   }
 }))
 
-const NotificationBox: FunctionComponent<BoxProps> = (props) => {
-  const { notifications } = useContext(ConfigContext)
-  const { currentTeamId } = useContext(TeamContext)
-
-  const notification = useMemo(() => {
-    const notification = get(notifications, [currentTeamId])
-    if (notification === undefined) {
-      return get(notifications, ['.default'])
-    }
-    return notification
-  }, [notifications, currentTeamId])
+const ClusterNotificationBox: FunctionComponent<{ cluster: any, setAmlLink: any}> = ({ cluster, setAmlLink }) => {
+  const { data } = useFetch(`/api/clusters/${cluster.id}`, [cluster.id])
+  const notifications = useMemo(() => {
+    if (data === undefined) return []
+    if (!Array.isArray(data.notifications)) return []
+    if (data.amlPortal !== undefined) setAmlLink(data.amlPortal)
```

Comment:
Do we assume `amlPortal`s are all the same?

Reply:
If all the same, better to make amlPortal something boolean and put the link to frontend configuration.